sql/2022/page-weight/page_weight_trend.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+SELECT
+  REPLACE(SUBSTR(_TABLE_SUFFIX, 0, 10), '_', '') AS date,
```

Comment:
Not sure why but YYYY-MM-DD format seems to work best for custom date formatting in sheets, ie to display as "Aug 2022".
```suggestion
  REPLACE(SUBSTR(_TABLE_SUFFIX, 0, 10), '_', '-') AS date,
```